.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*  @rakeshprabhu @harish-talview @merlano17 @devang1281 @manquer @KeerthiHarish
+*  @rakeshprabhu @harish-talview @merlano17 @devang1281 @KeerthiHarish
```

Comment:
Removing @manquer from code owner as part of https://github.com/talview/moodle-local_proview/pull/70#issuecomment-1716893995